pkgs/servers/sql/mariadb/default.nix

```diff
@@ -98,28 +96,26 @@ common = rec { # attributes common to both builds
 client = stdenv.mkDerivation (common // {
   name = "mariadb-client-${common.version}";
 
-  outputs = [ "bin" "dev" "out" ];
+  outputs = [ "dev" "out" "man" ];
```

Comment:
This is a non-standard output order; I would expect `"out"` to be first in this set.  https://nixos.org/nixpkgs/manual/#multiple-output-file-binaries-first-convention